congratulations/Congratulations.py

```diff
@@ -0,0 +1,52 @@
+from PIL import ImageTk, Image
+import webbrowser as webbrowser
+from urllib.parse import quote
+try:
+    import Tkinter as Tk        # Python < 3.0
+except ImportError:
+    import tkinter as Tk        # Python >= 3.0
+
+class Congratulations(object):
+    """Represents the completion of the goal. Houses functionality
+    for displaying a congratulatory message and prompting the user
+    for further action.
+
+    TO DO:
```

Comment:
Nice, that would be a useful feature. Do you wanna open up an issue with this suggestion, for the future? :)

Reply:
Yeah, but I was thinking about how to accomplish this and I think the first step has to be to represent the user as an object so that it's a bit easier to interact with it. Once I clean up that issue I'll probably come back and try to implement this.

Reply:
Yeah, user as an object sounds like a good suggestion, I hadn't considered it.